llvm/test/CodeGen/AArch64/avg-i128.ll

```diff
@@ -0,0 +1,126 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=aarch64 < %s | FileCheck %s
```

Comment:
are you intending to work on other midpoint / avg i128 handling? If not you should rename these avgflooru-i128.ll

Reply:
Alright, I'll make sure to fix that

Reply:
fixed